Dockerfile

```diff
@@ -2,17 +2,23 @@ FROM python:3.11-slim
 
 WORKDIR /app
 COPY ./requirements.txt /app
+
+ENV BUILD_DEPS="libssl-dev cargo gcc g++ libffi-dev"
```

Comment:
this "Just" adds another layer which makes the image slightly slower to download. is there an usecase for it?

Reply:
There was a reason during development, but there isn't really anymore.